src/base-client.ts

```diff
@@ -17,7 +17,9 @@ export class BaseSlackAPIClient implements BaseSlackClient {
   async apiCall(
     method: string,
     data: SlackAPIMethodArgs = {},
-  ): Promise<BaseResponse> {
+    //since we don't specifically know what the shape of the response object will be so set it to any.
+    // deno-lint-ignore no-explicit-any
+  ): Promise<any> {
```

Comment:
This should be done by allowing unexpected key value pairs within the `BaseResponse`, not changing the type that `apiCall` returns